src/helix-ui/elements/HXFileTileElement.js

```diff
@@ -67,40 +116,53 @@ export class HXFileTileElement extends HXElement {
     }
 
     /**
-     * File name
+     * @default false
+     * @readonly
+     * @type {Boolean}
+     */
+    get loading () {
+        return this.hasAttribute('progress');
+    }
+
+    /**
+     * File name to display
      * @type {String}
      */
     get name () {
         return this.getAttribute('name');
     }
     set name (newVal) {
-        this.setAttribute('name', newVal);
+        if (newVal === null) {
+            this.removeAttribute('name');
+        } else {
```

Comment:
Remove nullification logic. This was left over from an experiment that didn't pan out.